docker/release.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Usage:
+# // To build and publish the Docker image to the internal container registry.
+# $ DESTINATION=internal INTERNAL_CONTAINER_REGISTRY=<internal_registry> ./release.sh
+# // To publish the Docker image to the public container registry.
+# $ DESTINATION=public ./release.sh
+set -ex
+
+PUBLIC_CONTAINER_REGISTRY='gcr.io/stackdriver-agents/stackdriver-logging-agent'
```

Comment:
Can we convert single quotations to double quotations to be consistent throughout?